packetinjector/server_fake.go

```diff
@@ -0,0 +1,58 @@
+// +build !packetinject
```

Comment:
I would prefer to call it like the other "non implemented" files: no_server.go
https://github.com/skydive-project/skydive/blob/master/flow/no_ebpf.go

I would add something to avoid request from analyzer to agent so that the client side will get the error at the request time and the user will see the error. Maybe here replacing the validateRequest function

https://github.com/skydive-project/skydive/blob/master/api/server/packet_injector.go#L58

What do you think ?

Reply:
Renamed the file
Added changed to analyzer so an error is displayed on the UI side